runtime/jcl/common/mgmtthread.c

```diff
@@ -1344,6 +1378,14 @@ getThreadInfo(J9VMThread *currentThread, J9VMThread *targetThread, ThreadInfo *i
 
 	/* this may block on vm->managementDataLock */
 	getContentionStats(currentThread, targetThread, info);
+#if JAVA_SPEC_VERSION >= 19
+	continuation = targetThread->currentContinuation;
+	if (NULL != continuation) {
+		memcpy(&stackThread, targetThread, sizeof(J9VMThread));
```

Comment:
A comment should be added to describe why this step is needed. Also, why don't we do this step early?

Reply:
`getStackFramePCs` need to run on the correct java stack to produce the right output which is why we need to add this step. 
> Also, why don't we do this step earlier?

I wasn't sure if `getVMThreadObjectState/getJclThreadState/getContentionStats` need the vmthread address as a way of verifying object owner, so to avoid any potential issues, this step is done just before `getStackFramePCs`

Reply:
Also, include the above reason in the comment.